src/games/strategy/sound/ClipPlayer.java

```diff
@@ -419,16 +423,91 @@ private void parseClipPaths(final String pathName, final boolean subFolder) {
       thisSoundFile = null;
     }
 
+    if (thisSoundFile == null || !thisSoundFile.exists()) {
```

Comment:
Chunk of code that was removed in:
https://github.com/triplea-game/triplea/pull/201/files

Initial thought is this would find zip files of sounds, potentially inside of other zip files. Adding it back in part measure defensively, in part because it's likely still used somehow.